lib/matplotlib/axes/_axes.py

```diff
@@ -372,8 +372,8 @@ def legend(self, *args, **kwargs):
             The font properties of the legend. If None (default), the current
             :data:`matplotlib.rcParams` will be used.
 
-        fontsize : int or float or {'xx-small', 'x-small', 'small', 'medium',\
```

Comment:
That removes the newline from the string, and is necessary so that Sphinx interprets it correctly.